src/main/java/net/apnic/rdap/resource/ResourceConfiguration.java

```diff
@@ -0,0 +1,57 @@
+package net.apnic.rdap.resource;
+
+import net.apnic.rdap.autnum.AutnumStatsResourceLocator;
+import net.apnic.rdap.autnum.AsnRange;
+import net.apnic.rdap.domain.Domain;
+import net.apnic.rdap.domain.DomainStatsResourceLocator;
+import net.apnic.rdap.entity.Entity;
+import net.apnic.rdap.ip.IPStatsResourceLocator;
+import net.apnic.rdap.nameserver.NameServer;
+
+import net.ripe.ipresource.IpRange;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class ResourceConfiguration
+{
+    private ResourceLocator<AsnRange> autnumResourceLocator =
```

Comment:
Also makes testing a bit easier as you can swap out the ipResourceLocator bean with a test bean to poke & prod as needed/wanted.